### PR TITLE
multiple-choice widget screenshot test

### PR DESCRIPTION
**Pull request by HegePI**

### CODE REVIEW

File: system-tests/src/tests/latex.spec.ts [hunk 1]
```diff
@@ -126,8 +126,15 @@ test("latex-block renders", async ({ headless, page }) => {
   // Click text=Chapter 1: first page
   await Promise.all([page.waitForNavigation(), page.click("text=first page")])
   expectPath(page, "/courses/latex-course/chapter-1")
-  await expectScreenshotsToMatchSnapshots(page, headless, "latex", [
-    "text=Inline latex",
-    "text=Wubba Lubba Dub Dub",
-  ])
+  // await expectScreenshotsToMatchSnapshots(page, headless, "latex", [
```
Comment (nygrenh) [inline system-tests/src/tests/latex.spec.ts:129]: Remove dead code

File: system-tests/src/tests/model-solution.spec.ts [hunk 2]
```diff
@@ -39,12 +39,20 @@ test.describe("Model solutions", () => {
 
     const stableElement = await frame.waitForSelector("text=a")
 
-    await expectScreenshotsToMatchSnapshots(
+    // await expectScreenshotsToMatchSnapshots(
```
Comment (nygrenh) [inline system-tests/src/tests/model-solution.spec.ts:42]: Remove dead code

File: system-tests/src/tests/model-solution.spec.ts [hunk 3]
```diff
@@ -84,11 +92,19 @@ test.describe("Model solutions", () => {
...
```
Comment (nygrenh) [inline system-tests/src/tests/model-solution.spec.ts:95]: Dead code

File: services/quizzes/src/components/widget/MultipleChoice.tsx [hunk 4]
```diff
@@ -54,6 +54,9 @@ const MultipleChoice: React.FunctionComponent<QuizItemComponentProps> = ({
         className={css`
           display: flex;
           flex-direction: row;
+          @media only screen and (max-width: 600px) {
```
Comment (nygrenh) [inline services/quizzes/src/components/widget/MultipleChoice.tsx:57]: Refactor to use mobile first css. Also use respondToOrLarger

File: services/quizzes/src/components/widget/MultipleChoice.tsx [hunk 5]
```diff
@@ -73,6 +76,9 @@ const MultipleChoice: React.FunctionComponent<QuizItemComponentProps> = ({
             flex: 2;
             flex-direction: row;
             justify-content: space-between;
+            @media only screen and (max-width: 415px) {
```
Comment (nygrenh) [inline services/quizzes/src/components/widget/MultipleChoice.tsx:79]: Same here

File: system-tests/src/tests/exercise-service.spec.ts [hunk 6]
```diff
@@ -65,14 +65,25 @@ test("can add and delete exercise service", async ({ page, headless }) => {
   await page.click('button:has-text("Create")')
   await page.waitForSelector("text=New exercise service")
 
-  await expectScreenshotsToMatchSnapshots(
+  // await expectScreenshotsToMatchSnapshots(
```
Comment (nygrenh) [inline system-tests/src/tests/exercise-service.spec.ts:68]: Dead code

**Review by nygrenh — CHANGES_REQUESTED** [review]

**Review by jolampi — APPROVED** [review]

**Pull request merged**